contracts/lease/src/api/open/mod.rs

```diff
@@ -153,6 +153,16 @@ impl PositionSpec {
         obj
     }
 
+    pub fn validate_open_amount(self, amount: LpnCoin) -> ContractResult<()> {
+        if amount.amount() < self.min_sell_asset.amount() {
```

Comment:
do not do this
do we know they are of the same currency? How do we compare then?